app/src/test/kotlin/com/intive/picover/images/repository/ImagesRepositoryTest.kt

```diff
@@ -36,15 +34,16 @@ class ImagesRepositoryTest : ShouldSpec(
 
 		should("downloaded URIs WHEN storage reference and uri is fetched successfully") {
 			runTest(testDispatcher) {
-				val listResult: ListResult = mockk {
-					every { items } returns listOf(storageItem)
-				}
 				val uriResult: Uri = mockk()
+
 				every { imageReference.listAll() } returns mockk {
-					coEvery { await() } returns listResult
-				}
-				every { storageItem.downloadUrl } returns mockk {
-					coEvery { await() } returns uriResult
+					coEvery { await().items } returns listOf(
```

Comment:
You can also merge `imageReference.listAll()` with `await().items` there is no need to create a redundant mockk object, so the final form would be 
```
coEvery { imageReference.listAll().await().items }
```
Please apply above suggestion to other tests

Reply:
Please see: https://github.com/Android-Guild/Picover/pull/322

Reply:
Done. Please continue the conversation in https://github.com/Android-Guild/Picover/pull/315 or Tag me. Thank you.